include/gateway_config.hrl

```diff
@@ -16,7 +16,6 @@
 -define(MINER_MEMBER_ADD_GW, "AddGateway").
 -define(MINER_MEMBER_ASSERT_LOC, "AssertLocation").
 -define(MINER_MEMBER_P2P_STATUS, "P2PStatus").
--define(MINER_MEMBER_ETH_ONLINE, "EthernetOnline").
```

Comment:
Not referenced anywhere.

Reply:
Right, we're not asking miner about ethernet, good catch